### PR TITLE
Documentation comments

### PR DESCRIPTION
**Pull request by stephenwade**

### CODE REVIEW

File: NetSparkle/NetSparkle.cs [hunk 1]
```diff
@@ -303,6 +313,8 @@ public Sparkle(string appcastUrl, Icon applicationIcon, SecurityMode securityMod
         /// </summary>        
         /// <param name="appcastUrl">the URL for the appcast file</param>
         /// <param name="applicationIcon">If you're invoking this from a form, this would be this.Icon</param>
+        /// <param name="securityMode">Sparkle Security mode</param>
+        /// <param name="dsaPublicKey">The DSA public key to verfiy the sigatures.</param>
```
Comment (Deadpikle) [inline NetSparkle/NetSparkle.cs]: Please fix typo~

File: NetSparkle/NetSparkle.cs [hunk 2]
```diff
@@ -293,7 +302,8 @@ public Sparkle(string appcastUrl, Icon applicationIcon, SecurityMode securityMod
         /// </summary>
         /// <param name="appcastUrl">the URL for the appcast file</param>
         /// <param name="applicationIcon">If you're invoking this from a form, this would be this.Icon</param>
-        /// <param name="dsaPublicKey">The dsa public key to verfiy the sigatures.</param>
+        /// <param name="securityMode">Sparkle Security mode</param>
+        /// <param name="dsaPublicKey">The DSA public key to verfiy the sigatures.</param>
```
Comment (Deadpikle) [inline NetSparkle/NetSparkle.cs]: Typos~

File: NetSparkle/NetSparkle.cs [hunk 3]
```diff
@@ -313,6 +325,8 @@ public Sparkle(string appcastUrl, Icon applicationIcon, SecurityMode securityMod
         /// </summary>        
         /// <param name="appcastUrl">the URL for the appcast file</param>
         /// <param name="applicationIcon">If you're invoking this from a form, this would be this.Icon</param>
+        /// <param name="securityMode">Sparkle Security mode</param>
+        /// <param name="dsaPublicKey">The DSA public key to verfiy the sigatures.</param>
```
Comment (Deadpikle) [inline NetSparkle/NetSparkle.cs]: I see all the copy pasta typos~

Comment (stephenwade) [inline NetSparkle/NetSparkle.cs]: all three fixed by 9f236a0 and ab0b53c

**Pull request merged**